### DISCUSSION TITLE
Branching Strategy

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#905] [General]
Based on community feedback, I introduced a "dev" branch yesterday ( in addition to the "master" branch ) and made it the default branch. The concept is:

- master branch has the Oqtane 2.0.0 code and will only accept PRs for serious bug fixes which may necessitate a 2.0.1 release
- dev also has the Oqtane 2.0.0 code but will be the active development branch for enhancements as we work toward a 2.1.0 release

Note that introducing this new dev branch adds complexity - especially for those of you in the community who are not fully comfortable with Git. There is no doubt that in the longer term it adds a a lot of benefit, however this extra complexity will be quite challenging for some people and I expect that it may result in the community contributions decreasing in the short term. This is an area where we will need to help one another - so if anyone has questions on how to manage multiple branches, please do not be embarrassed to post them.

**Comment by hishamco** [comment]
Plus to what @sbwalker said before I encourage those who have a pending PRs to retarget them to the `dev` branch
